src/Validator/LIVR/Rules/ListLength.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace uMotif\Validator\LIVR\Rules;
+
+final class ListLength
+{
+    const FORMAT_ERROR = 'FORMAT_ERROR';
+
+    public static function listLength($param1, $param2)
+    {
+        return function ($values) use ($param1, $param2) {
+            if (!isset($param1) || $param1 === '' || $param2 < $param1) {
+                return self::FORMAT_ERROR;
```

Comment:
Can you update this one to `ListLength` to the one below to `self` to keep it consistent please.